src/coreclr/jit/lowerarmarch.cpp

```diff
@@ -2774,7 +2774,7 @@ GenTree* Lowering::TryLowerAddSubToMulLongOp(GenTreeOp* op)
     if (!comp->opts.OptimizationEnabled())
         return nullptr;
 
-    if (!JitConfig.EnableHWIntrinsic())
+    if (!comp->compOpportunisticallyDependsOn(InstructionSet_ArmBase_Arm64))
```

Comment:
are there other places where we use `EnableHWIntrinsics` and should use `compOpportunisticallyDependsOn` instead?

Reply:
Nope, just these two places.